lib/src/showcase.dart

```diff
@@ -309,6 +317,8 @@ class Showcase extends StatefulWidget {
     this.onTargetDoubleTap,
     this.disableDefaultTargetGestures = false,
     this.tooltipPosition,
+    this.titleTextDirection,
```

Comment:
Text direction will only be used in tool-tip not in custom widget so don't pass it in constructor. Make it null after constructor initialization.